prow/config.yaml

```diff
@@ -216,6 +216,9 @@ presubmits:
         ports:
         - containerPort: 9999
           hostPort: 9999
+        resources:
+          requests:
+            memory: "6Gi"
```

Comment:
This is for k/federation not k/k. Did you mean for this to be `2Gi` or is this correct?

Reply:
`k/fed` is just moved out of `k/k`, to be safe I set that matches rest of the k/k jobs, we can always update that later.